### ISSUE TITLE
Combo points showing despite disabled option

### ISSUE THREAD

**Issue by Gaviin1242** [Binbwen/tidyplates#18]
On my rogue, despite the Show Combo Points option under Other Widgets being unchecked, combo points still show up above my enemy nameplates when enemies are targeted.  They do not show up on untargeted mobs' nameplates.

Running 6.19.3.

**Comment by Binbwen** [comment]
Interesting!  I'll check it out; Thanks for the report

**Comment by Gaviin1242** [comment]
Thanks.  :)  One quick added detail... same thing seems to happen on my ret paladin and frost dk.  Holy power and frost runes show up above targeted enemy nameplates as well.

**Comment by Binbwen** [comment]
Ah, now that is interesting because I haven't yet implemented Holy Power or Runes in the widget...   TidyPlatesContinued HAS implemented those additional features, so that might cause some confusion out there.   Might also be another addon running? 

If you could drop a screenshot into this thread, that might help.

**Comment by Gaviin1242** [comment]
I don't think it's another addon, mainly because I haven't added anything new lately, and none of them really have anything to do with nameplates.  I'm beginning to wonder if this is a new Blizzard feature that they've baked into default nameplates as of 8.1, and they're just showing up on top of tidyplates.  Doesn't seem to be an option for it in interface settings, so I'm not sure... but they seem sort of Blizzard-y, with the look and feel of each being exactly as they are up where they normally are at the character pane.

Attached two screenshots from the rogue.  Let me know if you want to see the paladin and dk ones.

![wowscrnshot_121318_113225](https://user-images.githubusercontent.com/41398679/49956920-07a77a80-fecc-11e8-9c8d-ab13d2ab5a79.jpg)

**Comment by Gaviin1242** [comment]
Well, false alarm.  I figured it out.  Lol.  Apparently I didn't look hard enough in the default interface settings.  There's a new (maybe?) option under Unit Nameplates -> Personal Resource Display called "Show Special Resources on Targets".  This was checked for me but also grayed out because Personal Resource Display was unchecked.  So I had to recheck that in order to uncheck the special resources option, and once I did that, the combo points went away.

Either that option is new or 8.1 checked it for me, because this was never a problem until Tuesday.  Very strange.  Anyway, sorry for the confusion!  Thanks for the responsiveness.  :)
